<p>Alfred H. Sturtevant hypothesized that the frequency at which linked genes become unlinked through crossing over (<strong>recombination frequencies</strong>; calculated from experiments similar to the one in the figure below) could be used to determine the distances between genes on a chromosome. He predicted that the farther apart two genes were on a particular chromosome, the higher the probability that crossing over would occur between them, and subsequently, a higher recombination frequency would be observed.&nbsp;</p>
<p>[ciscode|rev=1|tool=elmsmedia|item=3123|entity_type=node|render=display_mode|display_mode=image__lightboxed]</p>
<p>By considering a chromosome a linear sequence of genes, Sturtevant assigned each gene he was studying in fruit fly crosses a position on the chromosome using recombination frequencies. &nbsp;The figure below&nbsp;illustrates one of Sturtevant's genetic maps, where three genes that are linked to each other (body color (<em>b</em>), wing size (<em>vg</em>) and an eye color gene called cinnabar (<em>cn</em>)) are positioned based on recombination frequencies determined through test crosses. This type of genetic map is called a&nbsp;<strong>linkage map</strong>&nbsp;because it portrays the sequence of genes along a chromosome, but it does not give the precise location of the genes.</p>
<p>To determine the distance between two genes, Sturtevant divided the number of gametes with recombinant chromosomes by the total number of gametes observed. In the figure above, the recombination frequency between&nbsp;<em>cn</em>&nbsp;and&nbsp;<em>b</em>&nbsp;is 9%, and the recombination frequency between&nbsp;<em>cn</em>&nbsp;and&nbsp;<em>vg</em>&nbsp;is 9.5%. Therefore, crossovers between&nbsp;<em>cn</em>&nbsp;and&nbsp;<em>b</em>, and between&nbsp;<em>cn</em>&nbsp;and&nbsp;<em>vg</em>, are about half as frequent as crossovers between&nbsp;<em>b</em>&nbsp;and&nbsp;<em>vg</em>&nbsp;(17%). A map that places&nbsp;<em>cn</em>&nbsp;between&nbsp;<em>b</em>&nbsp;and&nbsp;<em>vg</em>&nbsp;(approximately half-way) is consistent with these observations. Sturtevant expressed the distance between genes in&nbsp;<strong>map units</strong>. By definition, one map unit (1 m.u.) is equivalent to a 1% recombination frequency. In honor of Morgan, one map unit, or a 1% frequency, is also called one centimorgan (cM).</p>
<div class="row"><div class="col s12 l6">[ciscode|rev=1|tool=elmsmedia|item=3125|entity_type=node|render=display_mode|display_mode=image__lightboxed]</div>
<div class="col s12 l6">[ciscode|rev=1|tool=elmsmedia|item=3127|entity_type=node|render=display_mode|display_mode=image__lightboxed]</div></div>
<p>Thus, using crossover data, Sturtevant and his coworkers mapped other&nbsp;<em>Drosophila</em>&nbsp;genes in linear arrays at particular genetic locations. The figure above&nbsp;shows an abbreviated genetic map of chromosome II in&nbsp;<em>Drosophila</em>.</p>
<p>As with many rules, there are exceptions. The maximum recombination frequency that can be calculated between two genes is 49%.&nbsp; Once two genes are 50 map units apart, or further, the number of recombinant offspring produced will&nbsp;be equal to the number of offspring with the parental phenotypes.&nbsp; These genes would appear to assort independently and may mistakenly be thought to be located on different chromosomes because they are far enough apart on their chromosome that linkage is not observed in genetic crosses. These genes are mapped by adding the recombination frequencies from crosses involving intermediate genes, and by determining the approximate distance of each gene from the intermediates. For example, the genes for black body and brown eyes are move than 50 map units apart, so if you performed a test cross you would see a 1:1:1:1 ratio of phenotypes in the offspring. &nbsp;However, the genes for vestigial wings and cinnabar eyes are between these two genes at a distance of less than 50 map units, and you can to crosses to determine the recombination frequency between those genes and either black body or brown eyes, allowing you to construct a linkage map.</p>
<p>Genes that are close enough together on the same chromosome that they never recombine (cross-over) are described as being <strong>completely linked</strong>. &nbsp;The map distance between completely linked genes is 0.</p>
<p>&nbsp;<object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" height="480" id="myFlashContent" width="640">&nbsp;<object data="https://voicethread.com/book.swf?b=2106429" height="480" type="application/x-shockwave-flash" width="640"></object></object></p>
<p><a href="https://courses.ed.science.psu.edu/biol110/entity_iframe/node/4596" target="_blank">Chromosome Behavior and Gene Linkage Part 2 VoiceThread Transcript</a></p>
<p></p>
<p><object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" height="480" id="myFlashContent" width="640"><object data="https://voicethread.com/book.swf?b=2106600" height="480" type="application/x-shockwave-flash" width="640"></object></object></p>
<p><a href="https://courses.ed.science.psu.edu/biol110/entity_iframe/node/4595" target="_blank">Chromosome Behavior and Gene Linkage Part 3 VoiceThread Transcript</a></p>
<p><object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" height="480" id="myFlashContent" width="640"><object data="https://voicethread.com/book.swf?b=2106614" height="480" type="application/x-shockwave-flash" width="640"></object></object></p>
<p><a href="https://courses.ed.science.psu.edu/biol110/entity_iframe/node/4594" target="_blank">Chromosome Behavior and Gene Linkage Part 4 VoiceThread Transcript</a></p>
<p></p>