<h2>Terms</h2>
<p>You should have a working knowledge of the following terms:</p>
<ul>
	<li>complete linkage</li>
	<li>genetic map</li>
	<li>genetic recombination</li>
	<li>linkage map</li>
	<li>linked genes</li>
	<li>map unit</li>
	<li>nucleotide</li>
	<li>recombinant chromosome</li>
	<li>recombination frequency</li>
</ul>
<h2>Introduction and Goals</h2>
<p>The relationship between gene segregation and meiosis is examined here. As you should now know, Mendel was able to observe independent assortment between different genes because they were located on different chromosomes (each of which assorts randomly during meiosis). We also mentioned that genes located on the same chromosome may co-segregate because they are linked. Next, you will learn why they may, or may not, co-segregate. Moreover, you will learn how the frequency at which they become unlinked can be used to construct a genetic map of each chromosome. By the end of this tutorial you should have a basic understanding of:</p>
<ul>
	<li>Why genes are linked</li>
	<li>What complete vs. incomplete linkage means</li>
	<li>How genes recombine via crossing-over during prophase I</li>
	<li>How crossover frequency can be used to map genes</li>
	<li>How meiotic recombination and independent assortment contribute to genetic variation</li>
</ul>
<h2>Historical Background</h2>
<p><span class="lightbox"><img alt="Figure 1. A.H. Sturtevant relaxing at Woods Hole in 1916." class="lightbox" src="https://online.science.psu.edu/sites/default/files/biol110/Tutorial%208%20Figure%201%20A.%20H.%20Sturtevant.jpg" style="width: 300px; height: 192px; margin: 10px;" title="Figure 1. A.H. Sturtevant relaxing at Woods Hole in 1916." /></span></p>
<p>While at Columbia University in 1909, a young undergraduate student (Alfred Henry Sturtevant) took a class from Dr. Thomas H. Morgan, the famous geneticist who first described the relationship between genes and chromosomes (see tutorial 7). Sturtevant decided to continue in this line of study, due to his life-long interest in genetics. Sturtevant&#39;s interest had been fostered by his work on inheritance patterns of coat color in horses because he had spent his youth tabulating the pedigrees of his father&#39;s horses and studying the passing of color traits across generations. Sturtevant approached the distinguished professor and asked for a research position in Morgan&#39;s laboratory.</p>
<p><span class="lightbox"><img alt="Figure 2. A. H. Sturtevant." class="lightbox" src="https://online.science.psu.edu/sites/default/files/biol110/Tutorial%208%20Figure%202%20A.%20H.%20Sturtevant.jpg" style="width: 300px; height: 278px; margin: 10px;" title="Figure 2. A. H. Sturtevant." /></span></p>
<p>Intrigued by Sturtevant&#39;s work on coat color inheritance (which he later encouraged Alfred to publish in the Biological Bulletin; c. 1910), Morgan offered Sturtevant a desk and research project, in what later became known as "Morgan&#39;s Fly Room." Morgan had recently begun working with a relatively new model system for genetic analysis, the fruit fly <em>Drosophila melanogaster</em>. Together with Calvin Bridges and H.J. Muller, Sturtevant became a member of Morgan&#39;s lab at a time when the focus of the lab was shifting away from the study of development and embryology in marine invertebrates to the genetics of <em>Drosophila</em>.</p>
<p><span class="lightbox"><img alt="Figure 3. A. H. Sturtevant" class="lightbox" src="https://online.science.psu.edu/sites/default/files/biol110/Tutorial%208%20Figure%203%20A.%20H.%20Sturtevant.jpg" style="width: 300px; height: 329px; margin: 10px;" title="Figure 3. A. H. Sturtevant" /></span></p>
<p>Sturtevant was fascinated with his work in the "Fly Room." Over the course of his undergraduate research he spent hours talking about inheritance with his colleagues. One night, after a lengthy discussion about a paper describing coat color inheritance in rabbits, Sturtevant deduced that genes were linked in a series. He was so energized about his idea that he stayed up late that evening constructing experiments that would later demonstrate his theory and provide the basis of linkage analysis. Sturtevant&#39;s work (as well as that of his labmates) was a key component to the demonstration of the chromosome theory of heredity published by Morgan&#39;s group. Sturtevant continued his research as a graduate student in the lab, where he published the first <strong>genetic map</strong> of a chromosome as part of his graduate dissertation in 1913. Still, chromosome theory would not be definitively proven until after the Second World War.</p>
<h2>Pairing at Prophase I</h2>
<p>Chromosomes are linear pieces of DNA that are highly folded and compacted (<a href="https://online.science.psu.edu/sites/default/files/biol110/tutorial-2-heredity-and-life-cycles." target="_blank" title="Tutorial 2 - Heredity and Life Cycles">DNA Packaging</a>). If all of the DNA pieces from each of your 46 chromosomes (23 pairs) were linked end to end, the DNA would stretch about 2 meters. Therefore, DNA molecules are highly organized structures compacted into the nucleus of a single cell.</p>
<p><span class="lightbox"><img alt="Figure 4. DNA Structure" class="lightbox" src="https://online.science.psu.edu/sites/default/files/biol110/Tutorial%208%20Figure%204%20DNA%20structure.jpg" style="width: 300px; height: 333px; margin: 10px;" title="Figure 4. DNA Structure" /></span></p>
<p>DNA is a double-stranded molecule containing two antiparallel strands of nucleotides arranged in the form of a double helix. The two strands are held together by hydrogen bonds and contain nitrogenous bases (<strong>nucleotides</strong>) paired in a complementary fashion that appear as "rungs" on a spiral ladder. The complementary nucleotides pair specifically, such that adenine and guanine nucleotides (A and G) pair with corresponding thymine and cytosine nucleotides (T and C), respectively; thus, A pairs with T, and G pairs with C. The figure shown here depicts this in two ways. The image on the left clearly shows the base-pairing rules, whereas the image on the right depicts the helical arrangement of the paired polynucleotide strands.</p>
<p>Homologous chromosomes have similar but nonidentical nucleotide sequences. Indeed, it is these subtle differences in nucleotide sequences that form the molecular identify of every alternative allele for a given gene. During prophase I, homologous chromosomes associate and pair together due to their similarity in structure, length, and gene sequence. This sets up a physical association that facilitates the exchange of genetic sequences (i.e., pieces of DNA) between homologous chromosomes, in a process termed crossing over (previously discussed here --link to crossing over flash video.</p>
<h2>Crossing Over Between Homologous Chromosomes</h2>
<p><span class="lightbox"><img alt="Figure 5. Crossing-over" class="lightbox" src="https://online.science.psu.edu/sites/default/files/biol110/Tutorial%208%20Figure%205%20Crossing%20over.jpg" style="width: 300px; height: 171px; margin: 10px;" title="Figure 5. Crossing-over" /></span></p>
<p>The pairing of homologous chromosomes at prophase I is different than the pairing that occurs between complementary strands of DNA. In a mechanism that is not completely understood, the DNA of nonsister chromatids becomes precisely aligned. When homologues pair, both a maternal copy and a paternal copy of genetic information line up against each other. The association is so intimate that the two homologues swap genetic material in a process called <strong>crossing over</strong>.</p>
<p>This process is a source of <strong>genetic recombination</strong> and produces <strong>recombinant chromosomes</strong>. That is, a piece of a maternal chromatid exchanges with a piece of the paternal chromatid on the nonsister chromatid. There can be multiple crossovers between nonsister chromatids. Furthermore, crossover configurations can occur in any combination and can lead to dramatically different outcomes. See crossing over.</p>
<p>Only two chromatids are involved in any single crossover, and crossing over occurs at different points along the chromosome. Every meiotic process does not involve crossing over in the same set of genes each time, but there is at least one crossover per homologous pair. Still, when crossing over is observed in a meiotic division, up to half of the products of that division result in a recombination of DNA between the homologues. This recombination means the homologues are now different than the parents&#39; chromosomes, therefore, each may carry a different set of genetic information.</p>
<h2>Crossing Over is Random</h2>
<p>Many factors affect crossing over, so the position on the chromosome where crossing over will occur is unpredictable. Crossing over is a random event based on chance. The location of the break points on the DNA sequence of the chromosomes are somewhat random, but the recombination frequency is relatively constant between homologous chromosomes. (For a given chromosome, N number of breaks will occur, but where they will occur is random.)</p>
<p><object align="" height="480" width="640"><param name="movie" value="http://psu.voicethread.com/book.swf?b=2106348" /><param name="wmode" value="transparent" /><embed align="" height="480" src="http://psu.voicethread.com/book.swf?b=2106348" type="application/x-shockwave-flash" width="640" wmode="transparent"></embed></object></p>
<p>The probability of crossing over between genes on a chromosome is dependent on the distances between the genes. This shouldn&#39;t surprise you because the greater the distance between two genes, the greater the chance a break will occur.</p>
<p>Genes that are located on the same chromosome and that tend to be inherited together are termed <strong>linked genes</strong> because the DNA sequence containing the genes is passed along as a unit during meiosis. The closer that genes reside on a particular chromosome, the higher the probability that they will be inherited as a unit, since crossing over between two linked genes is not as common.&nbsp; <strong>Completely linked</strong> genes are genes that are so close together on a chromosome that crossing-over never occurs between them.</p>
<p>Linked genes do not follow the expected inheritance patterns predicted by Mendel&#39;s Theory of Independent Assortment when observed across several generations of crosses. Usually, crossing over between nonsister chromatids will occur between genes when they are relatively far apart on the homologous chromosomes when pairing occurs. This results in the production of an equal number of nonrecombinant and recombinant chromosomes. Thus, the ratio of offspring produced from test crosses will be 1:1:1:1 (fully paternal, paternal-maternal recombinant, maternal-paternal recombinant, and fully maternal). When half of all offspring have recombinant chromosomes, a 50% frequency of recombination is observed. Recall that in a test cross, a 1:1:1:1 ratio indicates that the genes are unlinked. Therefore, unlinked genes may either reside on different chromosomes or reside far apart on the same chromosome.</p>
<p>When two genes are very close together on each homologue, break points for crossing over between the two genes will be rare and fewer recombinant chromosomes will be produced. Under this circumstance, a ratio that deviates from the usual 1:1:1:1 will be observed, indicating that the genes are linked. Thus, crossing over between two particular genes on the same chromosome can be used as an indirect indicator of the distance between the two genes.</p>
<p><img border="0" height="0" src="https://c.gigcount.com/wildfire/IMP/CXNID=2000002.0NXC/bT*xJmx*PTEzMDc3OTY5NDYwODAmcHQ9MTMwNzc5NzExMTE5MiZwPTIwNjQyMSZkPWIyMTA2MzQ4Jmc9MiZvPTk2NjhhNGRlZDA4/NDRmNzlhYTY1ZTI4N2RlOTA2Y2E4Jm9mPTA=.gif" style="visibility: hidden; width: 0px; height: 0px;" width="0" /></p>
<h1>Evidence for Linked Genes in Drosophila</h1>
<p><span class="lightbox"><img alt="Figure 6. Crossing over accounts for recombinant phenotypes." class="lightbox" src="https://online.science.psu.edu/sites/default/files/biol110/Tutorial%208%20Figure%206%20Crossing%20over%20and%20recombiinants.jpg" style="width: 300px; height: 302px; margin: 10px;" title="Figure 6. Crossing over accounts for recombinant phenotypes." /></span></p>
<p>This figure demonstrates a test cross between flies differing in two characters: body color (<em>b</em>) and wing size (<em>vg</em>). The females are heterozygous for both genes and their phenotypes are wild type, so they display gray bodies and normal wings (<em>b+</em> <em>b</em> and <em>vg+</em> <em>vg</em>). The males are homozygous recessive and express the mutant phenotypes for both characteristics, so they display black bodies and vestigial wings (<em>b</em> <em>b</em> and <em>vg</em> <em>vg</em>). When T.H. Morgan scored this particular experiment and classified the offspring according to phenotype, he found that the parental phenotypes were disproportionately represented among offspring. If the two characters were on different chromosomes and assorted independently, Morgan would have expected to see a ratio of recombinant phenotypes to parental phenotypes of 1:1:1:1. Yet Morgan&#39;s observation of disproportionate offspring led him to conclude that the genes for body color and wing size in <em>Drosophila</em> were usually transmitted together from parents to offspring because they were located on the same chromosome. Therefore, the black body color gene and the vestigial wing gene are linked. This means that the genetic location for these genes are found close to one another and on the same chromosome.</p>
<p><a href="http://www.biology.arizona.edu/mendelian_genetics/problem_sets/dihybrid_cross/10Q.html" rel="_blank">Problem 10</a>: Exceptions to the 9:3:3:1 ratio of offspring? - Go over this question from The Biology Project Website. You may recall having seen this problem previously when you learned about dihybrid crosses, but it is particularly applicable to the subject of this tutorial.</p>
<h2>Using Cross Over Frequencies to Map Genes</h2>
<p><span class="lightbox"><img alt="Figure 7. Calculating Recombination Frequencies." class="lightbox" src="https://online.science.psu.edu/sites/default/files/biol110/Tutorial%208%20Figure%207%20Recombinant%20Frequencies.jpg" style="width: 400px; height: 240px; margin: 10px;" title="Figure 7. Calculating Recombination Frequencies." /></span></p>
<p>Alfred H. Sturtevant hypothesized that the frequency at which linked genes become unlinked (<strong>recombination frequencies</strong>; calculated from experiments similar to the one in this figure) could be used to determine the distances between genes on a chromosome. He predicted that the farther apart two genes were on a particular chromosome, the higher the probability that crossing over would occur between them, and subsequently, a higher recombination frequency would be observed.</p>
<p><object align="" height="480" width="640"><param name="movie" value="http://psu.voicethread.com/book.swf?b=2106429" /><param name="wmode" value="transparent" /><embed align="" height="480" src="http://psu.voicethread.com/book.swf?b=2106429" type="application/x-shockwave-flash" width="640" wmode="transparent"></embed></object></p>
<p>By considering a chromosome a linear sequence of genes, Sturtevant assigned each gene he was studying in fruit fly crosses a position on the chromosome using recombination frequencies. This figure illustrates one of Sturtevant&#39;s genetic maps, where three genes that are linked to each other (body color (b), wing size (vg) and an eye color gene called cinnabar (cn)) are positioned based on recombination frequencies observed in test crosses. This type of genetic map is called a <strong>linkage map</strong> because it portrays the sequence of genes along a chromosome, but it does not give the precise location of the genes.</p>
<p><span class="lightbox"><img alt="Figure 8. Genetic Map" class="lightbox" src="https://online.science.psu.edu/sites/default/files/biol110/Tutorial%208%20Figure%208%20Genetic%20Map.jpg" style="width: 300px; height: 114px; margin: 10px;" title="Figure 8. Genetic Map" /></span></p>
<p>To determine the distance between two genes, Sturtevant divided the number of gametes with recombinant chromosomes by the total number of gametes observed. In the figure above, the recombination frequency between cn and b is 9%, and the recombination frequency between cn and vg is 9.5%. Therefore, crossovers between cn and b, and between cn and vg, are about half as frequent as crossovers between b and vg (17%). A map that places cn between b and vg (approximately half-way) is consistent with these observations. Sturtevant expressed the distance between genes in <strong>map units</strong>. By definition, one map unit (1 m.u.) is equivalent to a 1% recombination frequency. In honor of Morgan, one map unit, or a 1% frequency, is also called one centimorgan (cM).</p>
<p><span class="lightbox"><img alt="Figure 9. Genetic Map Dros II" class="lightbox" src="https://online.science.psu.edu/sites/default/files/biol110/Tutorial%208%20Figure%209%20Genetic%20Map%20Dros%20II.jpg" style="width: 300px; height: 171px; margin: 10px;" title="Fig. 1 Conjugation between two strains of E. coli cells.." /></span></p>
<p>Thus, using crossover data, Sturtevant and his coworkers mapped other <em>Drosophila</em> genes in linear arrays at particular genetic locations. This figure depicts an abbreviated genetic map of chromosome II in <em>Drosophila</em>.</p>
<p>As with many rules, there are exceptions. If genes on the same chromosome are far apart, crossovers between them can occur frequently and these genes can have a maximum recombination frequency of 50%. These genes would appear to assort independently and may mistakenly be thought to exist on different chromosomes because they are so far apart on their chromosome that linkage is not observed in genetic crosses. These genes are mapped by adding the recombination frequencies from crosses involving intermediate genes, and by determining the approximate distance of each gene from the intermediates.</p>
<h2>Genetic Recombination and Meiosis</h2>
<p><span class="lightbox"><img alt="Figure 10. Crossing over during meiosis." class="lightbox" src="https://online.science.psu.edu/sites/default/files/biol110/Tutorial%208%20Figure%2010%20Crossing%20over.jpg" style="width: 300px; height: 124px; margin: 10px;" title="Figure 10. Crossing over during meiosis." /></span></p>
<p>In sexual reproduction, the events involving chromosomes during gametogenesis are responsible for the majority of genetic variation that arises in offspring. Most of this variability takes place during the first meiotic division (meiosis I) and involves genetic recombination between maternal and paternal chromosomes and the independent assortment of different chromosomes. This figure depicts the consequence of two crossover events between homologous chromosomes. Note that two of the meiotic products are different from the parents&#39; chromosomes.</p>
<p><img border="0" height="0" src="http://c.gigcount.com/wildfire/IMP/CXNID=2000002.0NXC/bT*xJmx*PTEzMDc3OTcyMDk4NDAmcHQ9MTMwNzc5NzIxNDAyNSZwPTIwNjQyMSZkPWIyMTA2NDI5Jmc9MiZvPTk2NjhhNGRlZDA4/NDRmNzlhYTY1ZTI4N2RlOTA2Y2E4Jm9mPTA=.gif" style="visibility: hidden; width: 0px; height: 0px;" width="0" /></p>
<h1><object align="" height="480" width="640"><param name="movie" value="http://psu.voicethread.com/book.swf?b=2106600" /><param name="wmode" value="transparent" /><embed align="" height="480" src="http://psu.voicethread.com/book.swf?b=2106600" type="application/x-shockwave-flash" width="640" wmode="transparent"></embed></object></h1>
<h2>Summary</h2>
<p>This tutorial examined the consequences of gene linkage and how crossing over in meiosis can alter the co-segregation pattern of genes that are located on the same chromosome. Recall that during prophase I, homologous chromosomes pair up, gene-for-gene. Occasionally this pairing leads to breakage and rejoining of regions of homologous chromosomes. In other words, portions of a maternally derived chromatid can exchange and become physically connected with regions of a paternally derived chromatid (and vice-versa). Any gene located on either side of this junction will become unlinked. The absolute point where crossover occurs cannot be predicted with certainty (although some regions of the chromosome are more prone to this than others); however, the greater the distance that two genes are from one another on the same chromosome, the greater the likelihood that crossing over will occur.</p>
<p>If the probability for crossing over increases proportionately with the distance between two linked genes, then the frequency at which they become unlinked (due to crossovers) is an indirect measure of how far apart these genes are from one another. If you don&#39;t understand why this is so, you might find it helpful to draw a couple of cartoons of chromosomes with two hypothetical genes at different distances from one another. Draw one set with the two genes 1/2 inch apart, and the other set with them one inch apart. If, on average, a crossover occurs once every four inches, then 25% of the time the two genes located one inch apart will become unlinked. However, the two genes located 1/2 inch apart will only become unlinked 12.5% of the time.</p>
<p>The relationship between gene distance and recombination frequency can be used to map the relative location of a gene on a chromosome. Be sure that you understand the logics behind this process that allows geneticists to map the relative order of genes on a chromosome.</p>
<p><img border="0" height="0" src="http://c.gigcount.com/wildfire/IMP/CXNID=2000002.0NXC/bT*xJmx*PTEzMDc3OTczMzYyMjgmcHQ9MTMwNzc5NzMzODY4MiZwPTIwNjQyMSZkPWIyMTA2NjAwJmc9MiZvPTk2NjhhNGRlZDA4/NDRmNzlhYTY1ZTI4N2RlOTA2Y2E4Jm9mPTA=.gif" style="visibility: hidden; width: 0px; height: 0px;" width="0" /><img border="0" height="0" src="http://c.gigcount.com/wildfire/IMP/CXNID=2000002.0NXC/bT*xJmx*PTEzMDc3OTc*MjczNjkmcHQ9MTMwNzc5NzQzMDAwOCZwPTIwNjQyMSZkPWIyMTA2NjE*Jmc9MiZvPTk2NjhhNGRlZDA4/NDRmNzlhYTY1ZTI4N2RlOTA2Y2E4Jm9mPTA=.gif" style="visibility: hidden; width: 0px; height: 0px;" width="0" /><object height="480" width="640"><param name="movie" value="http://psu.voicethread.com/book.swf?b=2106614" /><param name="wmode" value="transparent" /><embed height="480" src="http://psu.voicethread.com/book.swf?b=2106614" type="application/x-shockwave-flash" width="640" wmode="transparent"></embed></object></p>